tester_app/app/views/hello_world/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="alert alert-info">Client Rendering</h1>
+<%= react_component("HelloWorldApp",
+                     @hello_world_props,
+                     generator_function: false,
```

Comment:
@justin808 doesn't make a difference in practice, we should look into this further.